.prod/on_deploy.sh

```diff
@@ -3,3 +3,4 @@
 python /app/manage.py migrate --noinput
 
 # Admin commands that need to be ran for each env can be added here
+python /app/manage.py loaddata berth_products.json
```

Comment:
Is this needed in both `on_deploy` and `on_initialize`? The other `loaddata` files are not loaded here.

Reply:
Yes. The `on_initalize` is only ran on the `review` environments, since both `staging` and `prod` have already been initalized. However, for fresh review environments it's useful to have some data loaded.

Once this is merged and the command runs on staging and prod, we can remove it from here.